### DISCUSSION TITLE
How to return random data from array?

### DISCUSSION THREAD

**Discussion by irvanmoses** [AutomaApp/automa#787] [Q&A] [answered]
Hello, I have an array like this :

![image](https://user-images.githubusercontent.com/74028306/189800424-cd0c5ea0-8af4-45ac-bc9d-ce4bcee6ebcc.png)
 
I want to fill the form with data from this array.
How to get/return 1 data randomly?

**Comment by Kholid060** [comment, marked as answer]
Use the [`$randData`](https://docs.automa.site/api-reference/reference-data.html#randdata-expression) function. For example, when you have a variable with an array as the value, you can write it like `{{ $randData([variables@varName]) }}`

**Reply by irvanmoses** [reply]
Thank you so much!
